javascript/src/controllers/carousel-controller.js

```diff
@@ -0,0 +1,43 @@
+import { Controller } from 'stimulus'
+import Glide from '@glidejs/glide'
+
+export class CarouselController extends Controller {
+  static values = {
+    type: String,
+    index: Number,
+    perView: Number,
+    autoplay: Boolean,
+    gap: Number,
+    breakpoints: Object,
+    peek: Object
+  }
+
+  connect () {
+    const options = {
+      type: this.typeValue || 'carousel',
```

Comment:
Hay que importar Glide dinamicamente para reducir el tamaño del bundle inicial de Javascript

```suggestion
  async connect () {
    const { default: Glide } = await import('@glidejs/glide')
    
    const options = {
      type: this.typeValue || 'carousel',
```